esignet-core/src/main/java/io/mosip/esignet/core/validator/ClientAdditionalConfigValidator.java

```diff
@@ -0,0 +1,77 @@
+package io.mosip.esignet.core.validator;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.networknt.schema.JsonSchema;
+import com.networknt.schema.JsonSchemaFactory;
+import com.networknt.schema.SpecVersion;
+import com.networknt.schema.ValidationMessage;
+import io.mosip.esignet.core.exception.EsignetException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.io.DefaultResourceLoader;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.ResourceLoader;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+import java.util.Set;
+
+@Slf4j
+public class ClientAdditionalConfigValidator implements
```

Comment:
This can be marked as Component -> @Component. So that both ObjectMapper and ResourceLoader can be autowired.